man/man1/pdfsizeopt.1

```diff
@@ -0,0 +1,185 @@
+.TH "PdfSizeOpt" "1" "27 May 2019" "0.2019-05-27" "User Manual"
+
+.SH NAME
+\fBpdfsizeopt\fR
+
+.SH DESCRIPTION
+PdfSizeOpt - PDF file size optimizer.
+
+.SH SYNOPSIS
+\fBpdfsizeopt\fR [<flag>...] <input.pdf> [<output.pdf>]
+
+.SH OPTIONS
+.TP
```

Comment:
Please remove the descriptions of options. Even though they are correct now, they may become obsolete over time, and pdfsizeopt doesn't have enough contributor time to keep it up-to-date. Please add something like this instead: See pdfsizept --help for options.